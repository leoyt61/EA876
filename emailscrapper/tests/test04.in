e_mail.mandemais@gmail.com manda email para e_mail-interessante@fee.unicamp.br
