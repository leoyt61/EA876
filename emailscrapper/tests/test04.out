e_mail.mandemais@gmail.com
e_mail-interessante@fee.unicamp.br
